### DISCUSSION TITLE
Github Actions not work!

### DISCUSSION THREAD

**Discussion by kosuke-zhang** [community/community#110635] [Actions] [answered]
### Select Topic Area

Bug

### Body

When I push dev to origin, actions not work as expected and it work well about an hour ago.

**Comment by Grinch321** [comment]
+1

**Comment by AidenJamesYt** [comment, marked as answer]
github actions is currently experiencing degraded performance, meaning it is down for some people. To stay updated on the status, [visit the GitHub Status Site](https://www.githubstatus.com/) for live status updates.
